cachelib/navy/common/Device.cpp

```diff
@@ -905,6 +1038,7 @@ IoContext* FileDevice::getIoContext() {
 
     // Create new context if on event base thread or useIoUring_ is enabled
     bool useIoUring = ioEngine_ == IoEngine::IoUring;
+    bool useFdpIoUringCmd = isFDPEnabled_;
```

Comment:
Why additional variable?

Reply:
I thought of readability; to emphasize why we are taking the io_uring_cmd path. Anyway it can be removed.